__tests__/Unit/Components/Tasks/Option.test.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { render, fireEvent } from '@testing-library/react';
-import Option from '@/components/tasks/TaskSearch/Suggestion/Option';
+import Option from '@/components/tasks/TaskSearchDev/Suggestion/Option';
```

Comment:
why is the folder named `TaskSearchDev`?
I see it holds the filterModal - can we make it a common one which can be reused across?

Reply:
TaskSearchDev denotes all the features which the Tasksearch bar provides behind the dev flag. For example, generating suggestions based on user input.